docs/make.jl

```diff
@@ -22,7 +22,6 @@ makedocs(
         ],
         "Optical" => [
             "Systems" => "systems.md",
-            "Emitters" => "emitters.md",
             "Emitters (NEW)" => "emitters_new.md",
```

Comment:
Can probably get rid of the (NEW) now 😃

Reply:
I was thinking that we could keep it up for at least one minor version (and maybe even list the old emitters as DEPRECATED) just to make it clear that we switched. But we're probably a small enough project that it doesn't matter. I'll get rid of it.